package.json

```diff
@@ -1,63 +1,66 @@
-{
-  "name": "openapi-clientlib",
-  "version": "4.5.0",
-  "engines": {
-    "node": ">=4"
-  },
-  "scripts": {
-    "dist": "npm run clean && npm run rollup",
-    "clean": "rimraf dist",
-    "rollup": "rollup -c",
-    "dist-watch": "nodemon --watch src --watch libs -e js --exec npm run rollup",
-    "test": "npm run lint && npm run jest",
-    "lint": "eslint .",
-    "lint:fix": "eslint . --fix",
-    "jest": "jest --coverage",
-    "jest:watch": "jest --watch",
-    "jest:debug": "node --debug-brk node_modules/jest/bin/jest --runInBand"
-  },
-  "jest": {
-    "verbose": true,
-    "transform": {
-      "^.+\\.jsx?$": "babel-jest"
-    },
-    "moduleDirectories": [
-      "node_modules",
-      "<rootDir>"
-    ],
-    "testEnvironment": "node",
-    "restoreMocks": true,
-    "coverageDirectory": "dist/coverage/"
-  },
-  "main": "dist/release/openapi-client.js",
-  "files": [
-    "dist",
-    "src"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/SaxoBank/openapi-clientlib-js.git"
-  },
-  "license": "Apache-2.0",
-  "devDependencies": {
-    "@babel/core": "7.2.0",
-    "@babel/preset-env": "7.2.0",
-    "babel-core": "7.0.0-bridge.0",
-    "babel-jest": "23.6.0",
-    "eslint": "5.9.0",
-    "eslint-import-resolver-node": "0.3.2",
-    "eslint-plugin-destructuring": "2.1.0",
-    "eslint-plugin-import": "2.14.0",
-    "eslint-plugin-jest": "22.1.2",
-    "eslint-plugin-promise": "4.0.1",
-    "eslint-plugin-switch-case": "1.1.2",
-    "jest": "23.6.0",
-    "mockdate": "2.0.2",
-    "nodemon": "1.18.7",
-    "protobufjs": "6.8.8",
-    "regenerator-runtime": "0.13.1",
-    "rimraf": "2.6.2",
-    "rollup": "0.67.4",
-    "rollup-plugin-babel": "4.0.3"
-  }
-}
+{
```

Comment:
Hmm not sure what happen here. Different new lines probably. Will check. Otherwise file is almost the same (only diff is one new library: long).

Reply:
I don't see a new library? its probably line endings